Tasks/UseDotNetV2/task.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
I left some comments about the experience on both tasks and a couple other small comments.

Generally i'm fine with the changes. Sync w/ Matt/Bryan about the experience. If you guys are in sync, then i'm good with the changes.

Reply:
We reviewed the experience on the "auth" sections and we're questioning some stuff.  So we're moving forward with the tasks but pulling the auth stuff for now.

Reply:
Agreed